src/main/resources/application.yml

```
@@ -1,2 +1,6 @@
 server:
   port: 8000
+
+# Hierarquia => OFF, FATAL, ERROR, WARN, INFO, DEBUG, TRACE
+logging.level:
+  br.com.camelcase: TRACE
```

🤔 Talvez seja me jogar na frente do trem bala em movimento mas, acha que vale poder ser definido em variável de ambiente com `TRACE` como fallback?

Algo como:
```suggestion
  br.com.camelcase: ${LOG_LEVEL:-TRACE}
```

Se achar válido, não esquecer de adicionar a variável no `docker-compose` (não precisa valor no momento, só pra mapeamento)

---

Feito